pkg/metrics/maker.go

```diff
@@ -44,6 +44,9 @@ var makerBestAskPriceMetrics = prometheus.NewGaugeVec(
 	}, []string{"strategy_type", "strategy_id", "exchange", "symbol"})
 
 func UpdateMakerOpenOrderMetrics(strategyType, strategyId, exchangeName, symbol string, submitOrders []types.SubmitOrder) {
+	if len(submitOrders) == 0 {
+		return
+	}
```

Comment:
when submitOrders = 0, we should reset metrics?

Reply:
I add this as an extra protection.
There is an early return at line 1212 in `strategy.go`.

Reply:
Since we do not reset metrics before, I don't feel like we need to reset metrics here.

Reply:
I see.
Working on it.

Reply:
Done.